Examples/drawing-tests.lisp

```diff
@@ -175,6 +180,26 @@
          (:skip ()
            :report (lambda (stream) (format stream "skip ~a" (drawing-test-name test)))))))
 
+
+(defun drawing-test-pdf (test &optional filename)
+  (let* ((test (if (stringp test) (gethash test *drawing-tests*) test))
+         (test-name (drawing-test-name test))
+         (filename (or filename (format nil "/tmp/~a.pdf"
+                                        test-name))))
+    (with-open-file (out filename :direction :output :if-exists :supersede
+                         :element-type '(unsigned-byte 8))
+      (clim-pdf:with-output-to-pdf-stream (stream out)
+        #+nil
+        (with-text-style (stream (make-text-style :sans-serif :roman :normal))
```

Comment:
Hmm... it's because that's still broken. Maybe we should fix that before we merge.